### ISSUE TITLE
Windows support broken due to gunicorn

### ISSUE THREAD

**Issue by BYK** [GoogleCloudPlatform/functions-framework-python#38]
#36 switched to Gunicorn which is not supported on Windows. So as of `1.4.0`, `functions-framework` is not usable on Windows.

**Comment by di** [comment]
@BYK, sorry for the trouble. Can you share exactly what error you're getting?

For now, you should be able to work around this with the `--debug` flag, which uses Flask's built-in HTTP server instead.

**Comment by BYK** [comment]
>For now, you should be able to work around this with the --debug flag, which uses Flask's built-in HTTP server instead.

I think that may need to be the default or needs to be advertised for local development scenarios.

That being said, I cannot even run `--help` as it imports gunicorn by default, which tries to import `fnctl` and fails there :)

**Comment by BYK** [comment]
Full error and stack trace:
```
$ functions-framework --help
Traceback (most recent call last):
  File "C:\<redacted>\Python38\lib\runpy.py", line 193, in _run_module_as
_main
    return _run_code(code, main_globals, None,
  File "C:\<redacted>\Python38\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\<redacted>\Scripts\functions-framework.exe\__main__.py", line
4, in <module>
  File "c:\<redacted>\lib\site-packages\functions_framework\_cli.py", lin
e 20, in <module>
    from functions_framework._http import create_server
  File "c:\<redacted>\lib\site-packages\functions_framework\_http.py", li
ne 15, in <module>
    import gunicorn.app.base
  File "c:\<redacted>\lib\site-packages\gunicorn\app\base.py", line 11, i
n <module>
    from gunicorn import util
  File "c:\<redacted>\lib\site-packages\gunicorn\util.py", line 9, in <mo
dule>
    import fcntl
ModuleNotFoundError: No module named 'fcntl'
```

**Comment by di** [comment]
Got it, sorry about that. In that case, you can pin to [`functions-framework==1.3.0`](https://pypi.org/project/functions-framework/1.3.0/) for now.

**Comment by BYK** [comment]
@di yup, that's what I did. Thanks! :)

**Comment by di** [comment]
Fixed in [`functions-framework==1.4.1`](https://pypi.org/project/functions-framework/1.4.1/).
